buildconfig/CMake/Packaging/launch_mantidworkbench.sh.in

```diff
@@ -19,6 +19,13 @@ if [ -n "${PYTHONPATH}" ]; then
     LOCAL_PYTHONPATH=${LOCAL_PYTHONPATH}:${PYTHONPATH}
 fi
 
+SETUP_TOOLS_VERS=`pip3 show setuptools | grep Version | cut -d. -f1 | cut -d' ' -f2`
```

Comment:
Can we use `@PYTHON_EXECUTABLE@ -m pip` to ensure we use the same python executable as the rest of the script?